sunpy/map/mapbase.py

```diff
@@ -25,7 +25,6 @@
 from sunpy.sun import constants
 from sunpy.coordinates import sun
 from sunpy.time import parse_time, is_time
-from sunpy.image.transform import affine_transform
 from sunpy.image.resample import reshape_image_to_4d_superpixel
```

Comment:
Do these imports not pull in `affine_transform` anyway?

Reply:
Nope; what moving this import really does is avoid having to import skimage.

Reply:
I figured, I was just surprised.